tests/test_random_seed.c

```diff
@@ -0,0 +1,27 @@
+#include <assert.h>
```

Comment:
Let's drop this test.  It will not be reliable (it's perfectly valid for two random numbers to be identical), and to make it compile & link properly would require exporting json_c_get_random_seed() as a public API, which we should not do.

Reply:
Okay, it served my purpose of making sure the new code worked.